jyulb/fileio/common/jyu_lattice_proxy.py

```diff
@@ -0,0 +1,123 @@
+from simphony.cuds.abstractlattice import ABCLattice
+from simphony.cuds.lattice import LatticeNode
+import numpy as np
+
+
+class JYULatticeProxy(ABCLattice):
+
+    """
+    A proxy lattice for accessing state data of a JYU-LB modeling engine.
+
+    Updates and queries of node data are relayed to external data storages.
+    Acknowledges only those CUBA keywords which are prescribed at the
+    initialization.
+
+    Parameters
+    ----------
+    name : str
+    type : str
+        Bravais lattice type (should agree with the base_vect below).
+    base_vect : D x float
+        defines a Bravais lattice (an alternative for primitive vectors).
+    size : tuple of D x size
+        number of lattice nodes (in the direction of each axis).
+    origin : D x float
+    data : dictionary
```

Comment:
I see this `data` (i.e. `self._data`) is a dictionary that contains just node-specific data that updated by the wrapper during the run method.  maybe this could be called `external_node_data`?

This could avoid any confusions  as we  have `data : DataContainer` on all the high-level cuds objects, although I see that this not the case for ABCLattice but I do see it is in the D1.3 class diagram.